src/matrix/mod.rs

```diff
@@ -156,10 +177,7 @@ impl MatrixClient {
         Ok(json_response)
     }
 
-    pub async fn join_room(
-        &mut self,
-        room_id: &str,
-    ) -> Result<protocol::RoomJoinResponse, Error> {
+    pub async fn join_room(&mut self, room_id: &str) -> Result<protocol::RoomJoinResponse, Error> {
```

Comment:
A lot of these function signatures are changing from rustfmt and I dont know why

Reply:
Potentially the codebase just wasn't `rustfmt`ed before. I've thought this for some other bits of code. Don't worry about it.

Reply:
Ill look into adding it to the ci